page/code-organization/feature-browser-detection.md

```diff
@@ -98,25 +85,13 @@ if ( Modernizr.canvas ) {
 }
 ```
 
-That's it. Easy.
+For more in-depth information on Modernizr, feel free to check out their [documentation](http://modernizr.com/docs/).
 
 ### Performance Considerations
 
 So, while the Modernizr syntax is great, it can end up being quite cumbersome to have several conditionals. Secondly, we're sending the code for both conditions to every browser, regardless if we'll need it or not.
 
-The `Modernizr` object exposes a `load()` method that many prefer over the syntax mentioned previously. This is due to another library that Modernizr now uses internally: [yepnope](http://yepnopejs.com/). Testing for canvas can now become something like this:
-
-```
-Modernizr.load({
-	test: Modernizr.canvas,
-	yep: "canvas.js",
-	nope: "canvas-polyfill.js"
-});
-```
-
-Using the `load` method allows us to send only the required polyfills and code over the wire. You can also pass an array of objects as an argument to `.load()`, and it will serve the correct files to the correct audience.
-
-Additionally, Modernizr has a [production build configurator](http://modernizr.com/download/) that allows you to specify exactly what parts of Modernizr you want to include and exclude the parts you don't.
+If you're using Modernizr, we highly encourage you to use the [build configurator](http://modernizr.com/download/), a tool that allows you to create custom builds of the library. You can exclude checks you don't need, which might save you quite a bunch of bytes.
```

Comment:
"...which will save bytes and reduce the time it takes the page to load. Running every check that Modernizr can do, even when you don't need them, can slow down the page load."